static/app/views/replays/detail/layout/utils.tsx

```diff
@@ -0,0 +1,14 @@
+import theme from 'sentry/utils/theme';
+import {LayoutModes} from 'sentry/views/replays/detail/layout';
+
+export const getDefaultLayout = (): LayoutModes => {
+  const {innerWidth, innerHeight} = window;
+  const sidebarWidth = parseInt(theme.sidebar.expandedWidth, 10);
+  const windowsWidth = innerWidth - sidebarWidth;
+
+  if (windowsWidth < 1024 || innerHeight > 950) {
```

Comment:
> Can we do something that considers the height as well? I'm really curious to see how the proportion of the window can affect things.
> 
> 1024 seems reasonable on my laptop screen, but with different screen resolutions on external screens, ie: 2K, 4k I think it's not flexible enough to really change the output that people see.

Good catch, this is a thing I did not consider before - So I did some tests with a 2k monitor but only 14 inches and a 1080p monitor of 24 inches, and imo I belive that if you have less than this width is always more comfortable using the topbar layout. But if you have more than 950px of height, it felt more comfortable to use the topbar layout for things like the DOM tab. 

Let me know what you guys think!